\

```diff
@@ -0,0 +1,4 @@
+## feature/config
```

Comment:
Please, look at the file name.

Reply:
It happens with vim `:w` missclick, removed.